src/services/pdf-styles.js

```diff
@@ -125,7 +125,7 @@ const createFsaRnBox = (fsarnNumber, lcInfo) => {
   fsaRnBox.push({
     text: [
       {
-        text: "The unique food business registration number is",
```

Comment:
if yuo search for "reference number", you'll finda couple more instances to change in the readme and a tascomi test